packages/frontend/package.json

```diff
@@ -1,47 +1,49 @@
-  "description": "theguardian.com rendering tier",
-  "main": "index.js",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "@guardian/guui": "2.0.0-alpha.1",
-    "@guardian/pasteup": "1.0.0-alpha.1",
-    "aws-sdk": "^2.340.0",
-    "compose-function": "^3.0.3",
-    "curlyquotes": "^1.3.2",
-    "dateformat": "^3.0.3",
-    "diskusage": "^0.2.5",
-    "dompurify": "^1.0.3",
-    "emotion": "^9.1.1",
-    "emotion-server": "^9.1.1",
-    "express": "^4.16.4",
-    "glob": "^7.1.3",
-    "html-minifier": "^3.5.14",
-    "jsdom": "^13.0.0",
-    "lodash.get": "^4.4.2",
-    "minify-css-string": "^1.0.0",
-    "ophan-tracker-js": "^1.3.13",
-    "pm2": "^3.2.2",
-    "polished": "^1.9.2",
-    "react": "^16.4.0",
-    "react-dom": "^16.4.0",
-    "react-emotion": "^9.1.3"
-  },
-  "devDependencies": {
-    "@types/amphtml-validator": "^1.0.0",
-    "@types/compose-function": "^0.0.30",
-    "@types/dateformat": "^1.0.1",
-    "@types/dompurify": "^0.0.31",
-    "@types/express": "^4.16.0",
-    "@types/gapi.analytics": "^0.0.2",
-    "@types/google.analytics": "^0.0.39",
-    "@types/html-minifier": "^3.5.2",
-    "@types/jsdom": "^11.0.6",
-    "@types/lodash.get": "^4.4.4",
-    "@types/node": "^10.12.0",
-    "@types/react": "^16.4.11",
-    "@types/react-dom": "^16.0.7",
-    "amphtml-validator": "^1.0.23"
-  }
+    "name": "@guardian/frontend-rendering",
+    "version": "0.1.0-alpha.1",
+    "description": "theguardian.com rendering tier",
+    "main": "index.js",
+    "license": "Apache-2.0",
+    "dependencies": {
+        "@guardian/guui": "2.0.0-alpha.1",
+        "@guardian/pasteup": "1.0.0-alpha.1",
+        "aws-sdk": "^2.340.0",
+        "compose-function": "^3.0.3",
+        "curlyquotes": "^1.3.2",
+        "dateformat": "^3.0.3",
+        "diskusage": "^0.2.5",
+        "dompurify": "^1.0.3",
+        "emotion": "^9.1.1",
+        "emotion-server": "^9.1.1",
+        "express": "^4.16.4",
+        "glob": "^7.1.3",
+        "html-minifier": "^3.5.14",
+        "jsdom": "^13.0.0",
+        "lodash.get": "^4.4.2",
+        "minify-css-string": "^1.0.0",
+        "ophan-tracker-js": "^1.3.13",
+        "pm2": "^3.2.2",
+        "polished": "^1.9.2",
+        "raven-js": "3.19.1",
+        "react": "^16.4.0",
+        "react-dom": "^16.4.0",
+        "react-emotion": "^9.1.3"
+    },
+    "devDependencies": {
+        "@types/amphtml-validator": "^1.0.0",
+        "@types/compose-function": "^0.0.30",
+        "@types/dateformat": "^1.0.1",
+        "@types/dompurify": "^0.0.31",
+        "@types/express": "^4.16.0",
+        "@types/gapi.analytics": "^0.0.2",
+        "@types/google.analytics": "^0.0.39",
+        "@types/html-minifier": "^3.5.2",
+        "@types/jsdom": "^11.0.6",
+        "@types/lodash.get": "^4.4.4",
+        "@types/node": "^10.12.0",
+        "@types/raven-js": "^3.10.0",
+        "@types/react": "^16.4.11",
+        "@types/react-dom": "^16.0.7",
+        "amphtml-validator": "^1.0.23"
+    }
```

Comment:
Looks like the `package.json` now has 4 spaces instead of 2. We should fix this properly in the [`.editorconfig`](https://github.com/guardian/dotcom-rendering/blob/master/.editorconfig), can you fix manually for now?